WalletWasabi/Helpers/Constants.cs

```diff
@@ -18,10 +18,8 @@ public static class Constants
 
 		public const uint ProtocolVersionWitnessVersion = 70012;
 
-		public const int WitnessScaleFactor = 4;
-
 		public const int P2wpkhInputSizeInBytes = 41;
-		public const int P2wpkhInputVirtualSize = 68;
+		public const int P2wpkhInputVirtualSize = 69;
```

Comment:
This change is not dangerous because the constant is not being used by WW1.0

Reply:
cool, sorry for incorrectly saying that it was